conf/groups.conf.d/slurm.conf.example

```diff
@@ -18,8 +18,8 @@ reverse: sinfo -h -N -o "%R" -n $NODE
 [slurmstate,st]
 map: sinfo -h -o "%N" -t $GROUP
 all: sinfo -h -o "%N"
-list: sinfo -h -o "%T" | tr -d '*~#$@+'
-reverse: sinfo -h -N -o "%T" -n $NODE | tr -d '*~#$@+'
+list: sinfo -h -o "%T" | tr -d '*~#!%$@+^-'
```

Comment:
I hope Slurm supports UTF-8, as it will eventually exhaust us-ascii special characters... ;)

Wondering if you should not have a different strategy, where you filter only keeping the standard characters and remove all the other ones?

Reply:
I can't wait for emoji-based node states :)

But more seriously, I think the filtering should ideally *not* be based on what Slurm states strings could contain. but rather on what is considered acceptable characters in a group name, namely not those: https://github.com/cea-hpc/clustershell/blob/a1875cb78dc5f579e7053dae9bbcd848cc1c3a1f/lib/ClusterShell/NodeSet.py#L74

But I'm not sure we can get that illegal characters list in a group configuration file, can we?